katsdpimager/report.py

```diff
@@ -33,10 +33,21 @@
 
 
 FREQUENCY_PLOT_UNIT = u.MHz
-FLUX_PLOT_UNIT = u.Jy / u.beam
+FLUX_DENSITY_PLOT_UNIT = u.Jy
+PIXEL_PLOT_UNIT = u.Jy / u.beam
 ANGLE_PLOT_UNIT = u.deg
 PALETTE = bokeh.palettes.colorblind['Colorblind'][8]
-
+SI_FORMAT = bokeh.models.CustomJSHover(code="""
+    const threshold = [1e-9, 1e-6, 1e-3, 1e0, 1e3,  1e6,  1e9, 1e12];
+    const scale =     [1e12,  1e9,  1e6, 1e3, 1e0, 1e-3, 1e-6, 1e-9, 1e-12];
+    const suffix =    [ "p",  "n",  "µ", "m",  "",  "k",  "M",  "G", "T"];
```

Comment:
This made me start wondering what might be measured in Tera-Jy :)
And then I moved on to thinking about the 'pico-Jy' radio source population (not in our lifetimes) :)

Reply:
It's future-proof :-)

Reply:
Once you go beyond Tera-Jy are you into the femto-W/Hz/m^2 regime?